src/spec/client/services/product-projections.spec.coffee

```diff
@@ -65,6 +65,18 @@ describe 'ProductProjectionService', ->
   it 'should query for fuzzy', ->
     expect(@service.fuzzy()._queryString()).toBe 'fuzzy=true'
 
+  it 'should query for markMatchingVariants when set to true', ->
+    expect(@service.markMatchingVariants(true)._queryString()).toBe 'markMatchingVariants=true'
+
+  it 'should query for markMatchingVariants when set to false', ->
+    expect(@service.markMatchingVariants(false)._queryString()).toBe 'markMatchingVariants=false'
+
+  it 'should query for not set anything when no value is passed to markMatchingVariants', ->
+    expect(@service.markMatchingVariants()._queryString()).toBe ''
```

Comment:
At the API level ```markMatchingVariants``` is enabled by default.

As we want to disable it by default in the future at the API, I believe it is a good idea to start the change from the clients.

Therefore I vote for disabling by default in the SDK if possible in order to push actively the migration.

Check this for some context https://sphere.atlassian.net/wiki/display/S/Search+matching+variants+disabled+by+default